### PR TITLE
Add 2 new stupidly resistant wheelchairs.

### PR DESCRIPTION
**Pull request by Venuska1117**
- i dont know how to name wheelchairs better tbh.

## What Does This PR Do
Add 2 more types of wheelchairs, so disabled miners can melee rock on them during fights (they are not lava_proof), and add timer for crafting normal wheelchair.

## Why It's Good For The Game
Made you feel less worry that you will lose your wheelchair during ash drake fight and be vonurelable (or any other AoE boss, mainly made for lavaland miners)

## To Do:
- [x]  ADD ANY SPRITES

## Images:
image

## Testing
Compiled and it work.

## For lore team:

**Small Wheelchairs Lore:** 
_Our "outstanding" and "very smart" engineers decided during boredom to create new types of wheelchairs, because of their drunkness and clumsily, the blueprints they made were hard to follow, after they sobered it took them 6 hours to re-construct their wheelchair idea from clumsily made blueprints.

For sanity reasons, they made new blueprints with are more easy to follow._

|
**Hardened Wheelchair:**
Made from 15 plasteel, takes 6 seconds to make.

Made from a mixture of metal-plasma sheets, this wheelchair is 3 times stronger than a typical wheelchair and they are very resistant to acids and corrosion, thanks to that they can be used in various environments without the need to worry about safety.

|
**Reinforced Wheelchair:**
Made from 15 plastitanium, takes 7 seconds to make.

Made from a mixture of titanium-plasma sheets, this wheelchair is 6 times stronger than a typical wheelchair and they are very resistant to acids, corrosion and fire, thanks to that can be used in various environments, without the need to worry about safety... _but remember to not try bathe in lava._

## Changelog
:cl: Venuska1117
add: Add 2 more wheelchairs types, one made from plasteel and the other from plastitanium.
tweak: Add a timer for crafting a normal wheelchair.
/:cl:

### CODE REVIEW

**Comment by TravisAngeI** [comment]
what mad man is fighting megafauna in a wheelchair

**Comment by Venuska1117** [comment]
> what mad man is fighting megafauna in a wheelchair

if it get merged, maybe someone will be crazy enough to do it.

File: code/game/objects/structures/stool_bed_chair_nest/wheelchair.dm [hunk 1]
```diff
@@ -96,6 +96,30 @@
...
+
+/obj/structure/chair/wheelchair/plastitanium
+	name = "Reinforced Wheelchair"
+	desc = "Made from a mixture of titanium-plasma sheets, this wheelchair is 6 times stronger than a typical wheelchair and they are very resistant to acids, corrosion and fire, thanks to that can be used in various environments, without the need to worry about safety... <i>but remember to not try bathe in lava.</i>"
+	icon_state = "wheelchair"
+	item_chair = null
+	anchored = FALSE
+	movable = TRUE
+	max_integrity = 1500
+	buildstackamount = 15
```
Comment (DGamerL) [inline code/game/objects/structures/stool_bed_chair_nest/wheelchair.dm:119]: I think you can look at the base wheelchair type, and remove some of these duplicate variable declarations!

File: code/game/objects/structures/stool_bed_chair_nest/wheelchair.dm [hunk 2]
```diff
@@ -96,6 +96,30 @@
...
+	anchored = FALSE
+	movable = TRUE
+	max_integrity = 750
+	buildstackamount = 15
+	resistance_flags = ACID_PROOF
+	buildstacktype = /obj/item/stack/sheet/plasteel
+
+/obj/structure/chair/wheelchair/plastitanium
+	name = "Reinforced Wheelchair"
+	desc = "Made from a mixture of titanium-plasma sheets, this wheelchair is 6 times stronger than a typical wheelchair and they are very resistant to acids, corrosion and fire, thanks to that can be used in various environments, without the need to worry about safety... <i>but remember to not try bathe in lava.</i>"
```
Comment (DGamerL) [inline code/game/objects/structures/stool_bed_chair_nest/wheelchair.dm:113]:
```suggestion
	desc = "Made from a mixture of titanium-plasma sheets, this wheelchair is 6 times stronger than a typical wheelchair, and is very resistant to acid, corrosion and fire! Thanks to this, it can be used in various dangerous environments, without the need to worry about safety... <i>but remember to not try to bathe in lava.</i>"
```

File: code/game/objects/structures/stool_bed_chair_nest/wheelchair.dm [hunk 3]
```diff
@@ -96,6 +96,30 @@
 
 		occupant.visible_message("<span class='danger'>[occupant] crashed into \the [A]!</span>")
 
+/obj/structure/chair/wheelchair/plasteel
+	name = "Hardened Wheelchair"
+	desc = "Made from a mixture of metal-plasma sheets, this wheelchair is 3 times stronger than a typical wheelchair and they are very resistant to acids and corrosion, thanks to that they can be used in various environments without the need to worry about safety."
```
Comment (DGamerL) [inline code/game/objects/structures/stool_bed_chair_nest/wheelchair.dm:101]:
```suggestion
	desc = "Made from a mixture of metal-plasma sheets, this wheelchair is 3 times stronger than a typical wheelchair and is very resistant to acid and corrosion. Thanks to this it can be used in various dangerous environments without the need to worry about safety."
```

**Review by DGamerL — COMMENTED** [review]
First lookover

File: code/game/objects/items/stacks/sheets/sheet_types.dm [hunk 4]
```diff
@@ -37,7 +37,7 @@ GLOBAL_LIST_INIT(metal_recipes, list(
 		new /datum/stack_recipe("bench (right)", /obj/structure/chair/sofa/bench/right, 1, one_per_turf = TRUE, on_floor = TRUE),
 		)),
 	new /datum/stack_recipe("barber chair", /obj/structure/chair/barber, one_per_turf = TRUE, on_floor = TRUE),
-	new /datum/stack_recipe("wheelchair", /obj/structure/chair/wheelchair, 15, one_per_turf = TRUE, on_floor = TRUE),
+	new /datum/stack_recipe("wheelchair", /obj/structure/chair/wheelchair, 15, time = 5 SECONDS, one_per_turf = TRUE, on_floor = TRUE),
```
Comment (DGamerL) [inline code/game/objects/items/stacks/sheets/sheet_types.dm:40]: Was the wheelchair already 5 seconds to make?

**Review by DGamerL — COMMENTED** [review]
Seems quite solid, I just have one comment on it

File: code/game/objects/items/stacks/sheets/sheet_types.dm (hunk 4 above)
Comment (Venuska1117) [reply code/game/objects/items/stacks/sheets/sheet_types.dm:40]: no, it was instant craftable

File: code/game/objects/items/stacks/sheets/mineral.dm [hunk 5]
```diff
@@ -419,6 +419,7 @@ GLOBAL_LIST_INIT(titanium_recipes, list(
 GLOBAL_LIST_INIT(plastitanium_recipes, list(
 	new /datum/stack_recipe("plas-titanium tile", /obj/item/stack/tile/mineral/plastitanium, 1, 4, 20),
 	new /datum/stack_recipe("Kidan Warrior Statue", /obj/structure/statue/plastitanium/kidanstatue, 5, time = 2.5 SECONDS, one_per_turf = TRUE, on_floor = TRUE),
+	new /datum/stack_recipe("Reinforced Wheelchair", /obj/structure/chair/wheelchair/plastitanium, 15, time = 60 SECONDS, one_per_turf = TRUE),
```
Comment (Henri215) [inline code/game/objects/items/stacks/sheets/mineral.dm:422]: 60 seconds for reinforced and 20 for hardened? Thats not too much?

File: code/game/objects/structures/stool_bed_chair_nest/wheelchair.dm [hunk 6]
```diff
@@ -1,5 +1,5 @@
 /obj/structure/chair/wheelchair
-	name = "wheelchair"
+	name = "Wheelchair"
```
[the remark this answers is not in the document]
Comment (Venuska1117) [reply code/game/objects/structures/stool_bed_chair_nest/wheelchair.dm:2]: doesn't paradise prefer to capitalize first letters?

File: code/game/objects/items/stacks/sheets/mineral.dm (hunk 5 above)
Comment (Venuska1117) [reply code/game/objects/items/stacks/sheets/mineral.dm:422]: well they are intentionally this long as for wheelchairs, i can decrease time if its too much?

File: code/game/objects/structures/stool_bed_chair_nest/wheelchair.dm (hunk 6 above)
[the remark this answers is not in the document]
Comment (Contrabang) [reply code/game/objects/structures/stool_bed_chair_nest/wheelchair.dm:2]: Only really for proper names imo, objects like wheelchairs are improper nouns

File: code/game/objects/structures/stool_bed_chair_nest/wheelchair.dm [hunk 7]
```diff
@@ -96,6 +96,24 @@
...
+	name = "Hardened Wheelchair"
+	desc = "Made from a mixture of metal-plasma sheets, this wheelchair is 3 times stronger than a typical wheelchair and is very resistant to acid and corrosion. Thanks to this it can be used in various dangerous environments without the need to worry about safety."
+	icon_state = "h_wheelchair"
+	max_integrity = 750
+	buildstackamount = 15
+	resistance_flags = ACID_PROOF
+	buildstacktype = /obj/item/stack/sheet/plasteel
+
+/obj/structure/chair/wheelchair/plastitanium
+	name = "Reinforced Wheelchair"
```
Comment (Henri215) [inline code/game/objects/structures/stool_bed_chair_nest/wheelchair.dm:109]:
```suggestion
	name = "reinforced wheelchair"
```

File: code/game/objects/structures/stool_bed_chair_nest/wheelchair.dm [hunk 8]
```diff
@@ -96,6 +96,24 @@
 
 		occupant.visible_message("<span class='danger'>[occupant] crashed into \the [A]!</span>")
 
+/obj/structure/chair/wheelchair/plasteel
+	name = "Hardened Wheelchair"
```
Comment (Henri215) [inline code/game/objects/structures/stool_bed_chair_nest/wheelchair.dm:100]:
```suggestion
	name = "hardened wheelchair"
```

File: code/game/objects/items/stacks/sheets/sheet_types.dm [hunk 9]
```diff
@@ -161,6 +161,7 @@ GLOBAL_LIST_INIT(plasteel_recipes, list(
 	new /datum/stack_recipe("Surgery Table", /obj/machinery/optable, 5, time = 5 SECONDS, one_per_turf = TRUE, on_floor = TRUE),
 	new /datum/stack_recipe("Metal crate", /obj/structure/closet/crate, 10, time = 5 SECONDS, one_per_turf = TRUE),
 	new /datum/stack_recipe("Mass Driver frame", /obj/machinery/mass_driver_frame, 3, time = 5 SECONDS, one_per_turf = TRUE),
+	new /datum/stack_recipe("Hardened Wheelchair", /obj/structure/chair/wheelchair/plasteel, 15, time = 6 SECONDS, one_per_turf = TRUE),
```
Comment (Henri215) [inline code/game/objects/items/stacks/sheets/sheet_types.dm:164]:
```suggestion
	new /datum/stack_recipe("hardened wheelchair", /obj/structure/chair/wheelchair/plasteel, 15, time = 6 SECONDS, one_per_turf = TRUE),
```

File: code/game/objects/items/stacks/sheets/mineral.dm [hunk 10]
```diff
@@ -419,6 +419,7 @@ GLOBAL_LIST_INIT(titanium_recipes, list(
 GLOBAL_LIST_INIT(plastitanium_recipes, list(
 	new /datum/stack_recipe("plas-titanium tile", /obj/item/stack/tile/mineral/plastitanium, 1, 4, 20),
 	new /datum/stack_recipe("Kidan Warrior Statue", /obj/structure/statue/plastitanium/kidanstatue, 5, time = 2.5 SECONDS, one_per_turf = TRUE, on_floor = TRUE),
+	new /datum/stack_recipe("Reinforced Wheelchair", /obj/structure/chair/wheelchair/plastitanium, 15, time = 7 SECONDS, one_per_turf = TRUE),
```
Comment (Henri215) [inline code/game/objects/items/stacks/sheets/mineral.dm:422]:
```suggestion
	new /datum/stack_recipe("reinforced wheelchair", /obj/structure/chair/wheelchair/plastitanium, 15, time = 7 SECONDS, one_per_turf = TRUE),
```

**Comment by Henri215** [comment]
image
This can be removed from changelog.

File: code/game/objects/structures/stool_bed_chair_nest/wheelchair.dm [hunk 11]
```diff
@@ -96,6 +96,24 @@
...
+	buildstackamount = 15
+	resistance_flags = ACID_PROOF
+	buildstacktype = /obj/item/stack/sheet/plasteel
+
+/obj/structure/chair/wheelchair/plastitanium
+	name = "Reinforced Wheelchair"
+	desc = "Made from a mixture of titanium-plasma sheets, this wheelchair is 6 times stronger than a typical wheelchair, and is very resistant to acid, corrosion and fire! Thanks to this, it can be used in various dangerous environments, without the need to worry about safety... <i>but remember to not try to bathe in lava.</i>"
+	icon_state = "r_wheelchair"
+	max_integrity = 1500
+	buildstackamount = 15
```
Comment (Henri215) [inline code/game/objects/structures/stool_bed_chair_nest/wheelchair.dm:113]:
```suggestion
```
It inherits from base wheelchair

File: code/game/objects/structures/stool_bed_chair_nest/wheelchair.dm [hunk 12]
```diff
@@ -96,6 +96,24 @@
 
 		occupant.visible_message("<span class='danger'>[occupant] crashed into \the [A]!</span>")
 
+/obj/structure/chair/wheelchair/plasteel
+	name = "Hardened Wheelchair"
+	desc = "Made from a mixture of metal-plasma sheets, this wheelchair is 3 times stronger than a typical wheelchair and is very resistant to acid and corrosion. Thanks to this it can be used in various dangerous environments without the need to worry about safety."
+	icon_state = "h_wheelchair"
+	max_integrity = 750
+	buildstackamount = 15
```
Comment (Henri215) [inline code/game/objects/structures/stool_bed_chair_nest/wheelchair.dm:104]:
```suggestion
```
Same here

**Review by Henri215 — COMMENTED** [review]
Couldnt see any issues with the sprites so far

File: code/game/objects/items/stacks/sheets/sheet_types.dm (hunk 4 above)
Comment (Henri215) [inline code/game/objects/items/stacks/sheets/sheet_types.dm:40]: Other wheelchairs are missing the "on_floor = TRUE".
Hmm.. should a wheelchair need a floor to be made?

File: code/game/objects/structures/stool_bed_chair_nest/wheelchair.dm [hunk 13]
```diff
@@ -96,6 +96,22 @@
...
+	name = "hardened wheelchair"
+	desc = "Made from a mixture of metal-plasma sheets, this wheelchair is 3 times stronger than a typical wheelchair and is very resistant to acid and corrosion. Thanks to this it can be used in various dangerous environments without the need to worry about safety."
+	icon_state = "h_wheelchair"
+	max_integrity = 750
+	resistance_flags = ACID_PROOF
+	buildstacktype = /obj/item/stack/sheet/plasteel
+
+/obj/structure/chair/wheelchair/plastitanium
+	name = "reinforced wheelchair"
+	desc = "Made from a mixture of titanium-plasma sheets, this wheelchair is 6 times stronger than a typical wheelchair, and is very resistant to acid, corrosion and fire! Thanks to this, it can be used in various dangerous environments, without the need to worry about safety... <i>but remember to not try to bathe in lava.</i>"
```
Comment (Henri215) [inline code/game/objects/structures/stool_bed_chair_nest/wheelchair.dm:109]: One of the descriptions has a "," next to "typical wheelchair" while the other lacks it.
Also could try to avoid the "wheelchair" repetition.

File: code/game/objects/items/stacks/sheets/sheet_types.dm (hunk 4 above)
Comment (Venuska1117) [reply code/game/objects/items/stacks/sheets/sheet_types.dm:40]: so i should add it, right?

File: code/game/objects/structures/stool_bed_chair_nest/wheelchair.dm [hunk 14]
```diff
@@ -96,6 +96,22 @@
 
 		occupant.visible_message("<span class='danger'>[occupant] crashed into \the [A]!</span>")
 
+/obj/structure/chair/wheelchair/plasteel
+	name = "hardened wheelchair"
+	desc = "Made from a mixture of metal-plasma sheets, this wheelchair is 3 times stronger than classic model, and is very resistant to acid and corrosion. Thanks to this it can be used in various dangerous environments without the need to worry about safety."
```
Comment (Henri215) [inline code/game/objects/structures/stool_bed_chair_nest/wheelchair.dm:101]:
```suggestion
	desc = "Made from a mixture of metal-plasma sheets, this wheelchair is 3 times stronger than the classic model, and is very resistant to acid and corrosion. Thanks to this it can be used in hazardous environments without much worry."
```

File: code/game/objects/structures/stool_bed_chair_nest/wheelchair.dm [hunk 15]
```diff
@@ -96,6 +96,22 @@
...
+	icon_state = "h_wheelchair"
+	max_integrity = 750
+	resistance_flags = ACID_PROOF
+	buildstacktype = /obj/item/stack/sheet/plasteel
+
+/obj/structure/chair/wheelchair/plastitanium
+	name = "reinforced wheelchair"
+	desc = "Made from a mixture of titanium-plasma sheets, this wheelchair is 6 times stronger than classic model, and is very resistant to acid, corrosion and fire! Thanks to this, it can be used in various dangerous environments, without the need to worry about safety... <i>but remember to not try to bathe in lava.</i>"
```
Comment (Henri215) [inline code/game/objects/structures/stool_bed_chair_nest/wheelchair.dm:109]:
```suggestion
	desc = "Made from a mixture of titanium-plasma sheets, this wheelchair is 6 times stronger than the classic model, and is very resistant to acid, corrosion and fire! Thanks to this it can be used in hazardous environments without much worry... <i>but remember not to try to bathe in lava.</i>"
```

**Review by Henri215 — APPROVED** [review]
Looking good to me now after these suggestions

**Comment by Henri215** [comment]
image
I approved it, no need to request a review :D

**Comment by Venuska1117** [comment]
> image I approved it, no need to request a review :D

sorry :sadclown:

**Review by Contrabang — APPROVED** [review]
LGTM

**Review by DGamerL — APPROVED** [review]

**Review by S34NW — APPROVED** [review]
